lsp-completion.el

```diff
@@ -397,11 +398,11 @@ The MARKERS and PREFIX value will be attached to each candidate."
            result done?
            (candidates
             (lambda ()
+              ;; Force manual completion action so post completion logic can be run
+              (setq-local company--manual-action t)
               (lsp--catch 'input
                   (let ((lsp--throw-on-input lsp-completion-use-last-result)
-                        (same-session? (and (not done?)
```

Comment:
Why did you drop `(not done?)` and `(not result)`? Was it because they are always nil here?

Reply:
The check will not make much different since with the first `cond` clause is checking for `(or done? result)` already.
I'm planning to remove that cached `result` (company-capf is caching it) too if there's error.